### ISSUE TITLE
Error when bugzilla does not report flags

### ISSUE THREAD

**Issue by mikem23** [ralphbean/bugwarrior#535]
Lately, I've been getting this error when syncing from bugzilla:

```
ERROR:bugwarrior.services:Worker for [bz.redhat] failed: Bug object has no attribute 'flags'.
If 'flags' is a bugzilla attribute, it may not have been cached when the bug was fetched. You may want to adjust your include_fields for getbug/query.
Traceback (most recent call last):
  File "/home/mike/Devel/bugwarrior/build/lib/bugwarrior/services/__init__.py", line 506, in _aggregate_issues
    for issue in service.issues():
  File "/home/mike/Devel/bugwarrior/build/lib/bugwarrior/services/bz.py", line 257, in issues
    ) for bug in bugs
  File "/home/mike/Devel/bugwarrior/build/lib/bugwarrior/services/bz.py", line 256, in <genexpr>
    ((col, _get_bug_attr(bug, col)) for col in self.COLUMN_LIST)
  File "/home/mike/Devel/bugwarrior/build/lib/bugwarrior/services/bz.py", line 291, in _get_bug_attr
    return getattr(bug, attr)
  File "/usr/lib/python2.7/site-packages/bugzilla/bug.py", line 110, in __getattr__
    raise AttributeError(msg)
AttributeError: Bug object has no attribute 'flags'.
If 'flags' is a bugzilla attribute, it may not have been cached when the bug was fetched. You may want to adjust your include_fields for getbug/query.
INFO:bugwarrior.services:Done with [bz.redhat] in 7.013321s
```

There's been no change in the bugwarrior code I'm running, so either something changed in python-bugzilla, or with the particular bz instance I am connecting to (rh).

**Comment by mikem23** [comment]
I have a workaround, which I will PR shortly

**Comment by ryneeverett** [comment]
Resolved via #536.
